patches/net/minecraft/client/gui/components/EditBox.java.patch

```diff
@@ -0,0 +1,60 @@
+--- a/net/minecraft/client/gui/components/EditBox.java
++++ b/net/minecraft/client/gui/components/EditBox.java
+@@ -56,6 +_,7 @@
+     @Nullable
+     private Component hint;
+     private long focusedTime = Util.getMillis();
++    private boolean shadow = true;
```

Comment:
Would it be a good idea to add a comment explaining why we have this patch? Something along the lines of `// Neo: Allow modders to disable the default shadow.` 🤔